--- flooding.lua	2017-11-22 23:02:38.000000000 +0700
+++ flooding_edit.lua	2018-03-24 16:32:31.525985175 +0700
@@ -180,7 +180,7 @@
 
 function Flooding:OnSave()
 
-	local nextmode,currentmode = GetWorld().Flooding:GetModeData()
+	--local nextmode,currentmode = GetWorld().Flooding:GetModeData()
 
 	local data = {}
 	if self.inst.Flooding then
@@ -194,8 +194,8 @@
 		data.maxFloodLevel = self.maxFloodLevel
 		data.floodFrequency = GetWorld().Flooding:GetSpawnerFrequency()
 		data.timeSinceFloodIncrease = self.timeSinceFloodIncrease
-		data.nextmode = nextmode
-		data.currentmode = currentmode
+		--data.nextmode = nextmode
+		--data.currentmode = currentmode
 	end
 	return data
 end
@@ -210,9 +210,9 @@
 				data.currentmode = 0
 			end
 		end
-		if data.nextmode and data.currentmode then
-			GetWorld().Flooding:SetModeData(data.nextmode,data.currentmode)			
-		end
+		--if data.nextmode and data.currentmode then
+			--GetWorld().Flooding:SetModeData(data.nextmode,data.currentmode)			
+		--end
 end
 
 function Flooding:OnLoad(data)
